docs/architecture/adr-0014-api-documentaton-revisited.md

```diff
@@ -0,0 +1,104 @@
+
+#### Angular 1.X
+##### Pros
+* We know it already, so it would be quick to get up and running
+* Probably "good enough" for all intents and purposes. This is the quickest and easiest solution
+##### Cons
+* We are planning on eventually moving away from it
+* Angular 1.X libraries are kind of in limbo right now - We've found that quite 
+a few frameworks are no longer being actively maintained, so we have to bring the 
+code into our own repository for maintenance purposes.  New angular libraries will
+also likely only be written for angular 2+, possibly limiting our options down the road to
+projects which only update with bugfixes.
+* Not auto-generated
+
+#### Angular 2
+##### Pros
+* Lets us experiment with Angular 2 before we move decide to move one of our larger projects over
+* Aligns with team objectives
+* Performance is aligned with the other major frameworks
+* Angular CLI seems pretty solid.  I haven't run into any of the usual webpack issues using it.
+  There is now an RC.0, which means that no project code changes should be required in order to 
+  upgrade to v1
+* Typescript is amazing
+* Supports Redux
+* Html templates with javascript controllers
+* Maintained by Google: The direction that a central team provides can be valuable
+
+##### Cons
+* We need to learn the framework while we develop
+  * Ecosystem is still a bit young compared to Angular 1.X and Ember
+  * Need to learn typescript
+  * Many core concepts have changed from Angular 1.X (services, scope, etc). 
+    The framework is more like a plug and play setup for react with angular influence
+* Maintained by Google: Its success leans heavily on Google's continued support (see )
+* Not auto-generated
+
+
+#### Ember
+
+##### Pros
+* Existing prototype
+* Stable, open source framework that isn't going to disappear unless something
+  much better comes along and invalidates the work that's been done
+* Performance aligned with other mainstream frameworks
+* Supports typescript
+* awesome CLI
+* Supports Redux
+* Good ecosystem - well tested modules
+* Good upgrade tooling
+* html with inline javascript
+
+##### Cons
+* Learning ember - one more framework to keep track of
+* A classic open source project that is maintained by the community. 
+* Not auto generated
+
+## Proposed Solution
+We will write the api documentation as a traditional SPA using Angular 2, using either
+Webpack or Angular CLI as a build tool. This should allow us to accomplish our flexibility
+goals while also allowing us to test new technology in a place where it will have less impact.
+
+## Consequences / Tradeoffs
+### Libraries
+#### Pros
+Angular 2 solves a lot of issues that Angular 1.X has, and allows for cleaner, more maintainable code.
+Angular 2's design was highly influenced by the progress made in frameworks such as React and Ember,
+which addresses many of the issues that people had with Angular 1.X such as poor performance and
+highly complex controller and template logic
```

Comment:
Same point here; we're not evaluating Angular 2 only with respect to Angular 1.x, but also comparing it to React and Ember as well, so these should be included.